client/src/actions/index.js

```diff
@@ -17,44 +15,44 @@ function catchErrorsWrap(fn) {
   };
 }
 
-async function doRequestInitialize() {
-  const res = await fetch(
-    `${globals.API.prefix}${globals.API.version}initialize`,
-    {
-      method: "get",
-      headers: new Headers({
-        "Content-Type": "application/json"
-      })
-    }
-  );
-  return res.json();
-}
-
-async function doRequestCells(query) {
-  const res = await fetch(
-    `${globals.API.prefix}${globals.API.version}cells${query}`,
-    {
-      method: "get",
-      headers: new Headers({
-        "Content-Type": "application/json"
-      })
-    }
-  );
+/*
+Bootstrap application with the initial data loading.
+  * /config - application configuration
+  * /schema - schema of dataframe
+  * /annotations/obs - all metadata annotation
+*/
+const doJsonRequest = async url => {
```

Comment:
The name implies to me that this is general / generic and will be consumed by multiple other actions. If we intend on using this as a standalone ever, I would suggest renaming to something more specific.

Reply:
Ie., this could almost be broken out into a 'util' file as it does not look like a proper action

Reply:
I'm not up to speed on proper style in action creators.  Should there be (literally) no helper code in these files?   If so, I can move both this and `catchErrorsWrap` into a utils helper file.